index.ios.js

```diff
@@ -19,6 +19,7 @@ import thunk from 'redux-thunk';
 import createLogger from 'redux-logger';
 import promise from 'redux-promise';
 const SideMenu = require('react-native-side-menu');
+const Firebase = require('firebase');
```

Comment:
thanks for catching! This was done earlier by someone else so I forgot to delete it. :heavy_check_mark: